localization/languages/ko.json

```diff
@@ -199,5 +199,28 @@
         "downloadStateFailed": "실패함",
         /* Update Notifications */
         "updateNotificationTitle": "새롭게 판올림된 민(Min) 사용 가능"
+        /* Autofill settings */
+        "settingsPasswordAutoFillHeadline": "비밀번호 자동완성",
+        "settingsSelectPasswordManager": "지원하는 비밀번호 관리자 중에서 하나를 선택하세요:",
+        /* Password manager setup */
+        "passwordManagerSetupHeading": "자동완성을 사용하려면 %p 설정을 완료하세요.",
+        "passwordManagerSetupStep1": {
+            "unsafeHTML": "우선, <a id='password-manager-setup-link'></a>를 방문해 시스템에 내려받으세요."
+        },
+        "passwordManagerSetupLink": "%p CLI 도구를 내려받으세요."
+        "passwordManagerSetupStep2": "그 다음 도구를 아래 박스 안에 끌어다 놓습니다:",
+        "passwordManagerSetupDragBox": "도구를 여기에 끌어다 놓으세요.",
+        "passwordManagerSetupInstalling": "설치하는 중...",
+        "passwordManagerSetupSignIn": "자동완성을 사용하려면 비밀번호 관리자에 로그인하세요. 귀하의 자격증명은 어디에도 저장되지 않습니다.",
```

Comment:
Google Translate (which I realize isn't always right) is translating this as "Your credentials are not stored anywhere", which isn't accurate - Min itself doesn't save the credentials, but they're sent to your password manager, which could potentially save them (although it probably won't). Could you please confirm that the translation only refers to what Min will do, and not the credentials in general?

Reply:
Done.

Reply:
Thank you!